security/WFLY-14922_Remove_PicketLink_Subsystem.adoc

```diff
@@ -0,0 +1,125 @@
+= WFLY-14922 Remove PicketLink Subsystem from WildFly
+:author:            Farah Juma
+:email:             fjuma@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The PicketLink subsystem in WildFly has been deprecated for quite some time now. Related to this,
+setting up SSO with SAML using PicketLink is also deprecated. In order to configure SSO
+with SAML for WildFly, the
+https://www.keycloak.org/docs/latest/securing_apps/#jboss-eap-wildfly-adapter-2[Keycloak SAML Elytron adapter]
+can be used instead.
+
+This task is focussed on the removal of this subsystem from WildFly and the related migration path
+for configuring SSO with SAML for WildFly.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-14922[WFLY-14922 - Remove picketlink extension and subsystems]
+* https://issues.redhat.com/browse/WFLY-15076[WFLY-15076 - Add documentation on how to configure SSO with SAML for WildFly using the Keycloak adapter]
+* https://issues.redhat.com/browse/KEYCLOAK-18924[KEYCLOAK-18924 - The org.picketbox module dependency should be removed from the SAML Elytron adapter module.xml file]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1750[EAP7-1750 - Remove PicketLink Subsystem]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+* [x] Engineering
+
+* [x] QE
+
+As a removal RFE, this does not require new feature test development. However both teams will need
+to review existing testsuites for tests dependent on the PicketLink subsystem. Identified tests will
+either need to be deleted or they will need to be converted to use the new approach.
+
+=== Affected Projects or Components
+
+* WildFly
+* Keycloak
+
+=== Other Interested Projects
+
+* [x] External Testsuites
+
+* [x] Quickstarts
+
+=== Relevant Installation Types
+
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
+
+== Requirements
+
+Since this is a removal task, this section will describe the steps that will be needed as opposed to the
+set of requirements.
+
+=== Hard Requirements
+
+* The PicketLink subsystem needs to be converted to a skeleton subsystem. (It needs to be a skeleton subsystem
```

Comment:
I'm not sure about the expected behavior of a skeleton subsystem. Can you elaborate this please? Is the detailed configuration of the subsystem preserved in the new version of the management model? Can it be updated through management CLI operations? Will there be a warning in standalone mode? And the like.

Reply:
I've added more details in the document. Let me know if anything still isn't clear.

Reply:
It's clear now, thank you.